runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -103,16 +105,367 @@ void VMgenerateCatchBlockBBStartPrologue(TR::Node *node, TR::Instruction *fenceI
       }
    }
 
+/**
+ * @brief Generates instructions to load j9class from object pointer
+ *
+ * @param[in]       node: node
+ * @param[in] j9classReg: register j9class value is assigned to
+ * @param[in]     objReg: register holding object pointer
+ * @param[in]         cg: code generator
+ */
+static void
+generateLoadJ9Class(TR::Node *node, TR::Register *j9classReg, TR::Register *objReg, TR::CodeGenerator *cg)
+   {
+   generateTrg1MemInstruction(cg, TR::Compiler->om.compressObjectReferences() ? TR::InstOpCode::ldrimmw : TR::InstOpCode::ldrimmx, node, j9classReg,
+      new (cg->trHeapMemory()) TR::MemoryReference(objReg, static_cast<int32_t>(TR::Compiler->om.offsetOfObjectVftField()), cg));
+   TR::TreeEvaluator::generateVFTMaskInstruction(cg, node, j9classReg);
+   }
+
+/**
+ * @brief Returns opcode for load/store with immediate offset of specified data type
+ *
+ * @param[in] cg: CodeGenerator
+ * @param[in] dt: data type
+ * @param[in] elementSize: element size
+ * @param[in] isLoad : true if operation is load
+ * @return opcode
+ */
+static TR::InstOpCode::Mnemonic getLoadOrStoreFromDataType(TR::CodeGenerator *cg, TR::DataType dt, int32_t elementSize, bool isLoad)
+   {
+   switch (dt)
+      {
+   case TR::Int8:
```

Comment:
Please change the indentation of case labels in this switch statement like this.
```
   switch (dt)
      {
      case TR::Int8:
         return isLoad ? TR::InstOpCode::ldrbimm : TR::InstOpCode::strbimm;
      case TR::Int32:
         ...
         return TR::InstOpCode::bad;
      }
```